src/components/Projects/WBS/WBSDetail/AddTask/AddTaskModal.jsx

```diff
@@ -307,8 +305,22 @@ const AddTaskModal = props => {
     setEndstateInfo(props.tasks.copiedTask.endstateInfo);
   };
 
+  //FUNCTION TO UPDATE TASK MOTHER
+  const updateTaskMother = () => {
```

Comment:
Changes in this file are not related to this PR, please revert them. We should always follow "One PR only do (a part of) one thing" rule.